### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by C0deZLee** [GetBlimp/django-rest-framework-jwt#414]
It has not been updated since 3 months ago

**Comment by eskhool** [comment]
More importantly it has many urgent issues without any comments by the BDFL (whoever he/she may be)

**Comment by sphrak** [comment]
I am kinda wondering this aswell. Could we get a statement from the maintainers?

**Comment by vladislav-goncharenko** [comment]
Yeah, would be excellent to get response from @jpadilla on this...

**Comment by brainwane** [comment]
@jpadilla is the cofounder of [a company based in Puerto Rico](https://www.blimp.io/); he lives in Connecticut but I'm still guessing that he's dealing with a lot right now. Best wishes @jpadilla.

**Comment by aeab13** [comment]
I'd love to see some feedback from the developers. Imo this is a really great project, it'd be a shame to let it die.

**Comment by eskhool** [comment]
Yeah, its a great project but it doesn't have to die. What would be nice is for @jpadilla (thanks for the good work btw) can just update us on the status even if its to say I'm alive but won't be giving it more than 1 hr a year or something :) That can help folk make the decision to start forking or looking at other options.

**Comment by psychok7** [comment]
Also curious about the status on this one especially when starting new projects..

**Comment by mr337** [comment]
Sadly not hearing much and debating to add this to new projects :(

**Comment by valentinogagliardi** [comment]
Me too

**Comment by codeadict** [comment]
Seems like this is abandoned, I may for it and accept PRs but will see if the original author responds and is ok with that. Is sad if it dies.

**Comment by Alex3917** [comment]
I'm using it as a dependency for a bunch of projects. If it ever gets to the point where there are issues with the newest versions of Django or DRF then I'm happy to fix any issues. Right now though there doesn't seem to be anything broken, despite the fact that the newest versions of Django and DRF aren't in the test matrix.
